### PR TITLE
Upgrade c2c.template

### PR DESCRIPTION
**Pull request by sbrunner**
Update the vars files
Better ngeo dependency
Configurable container dependencies

### CODE REVIEW

**Comment by arnaud-morvan** [comment]
I do not understand why you've moved initialize_db to admin/acceptance_tests, this should be the prefered option to create a new database schema from scratch. It should be accessible in commons.

**Comment by sbrunner** [comment]
Because it's used only for the tests, it shouldn't be used on a relay environment...

File: commons/acceptance_tests/initializedb.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    dbsession.merge(User(
+        username='admin',
+        email='admin@camptocamp.com',
+        role=role_admin
+    ))
+    dbsession.merge(User(
+        username='user',
+        email='user@camptocamp.com',
+        role=role_user
+    ))
```
Comment (arnaud-morvan) [inline commons/acceptance_tests/initializedb.py]: Is this a duplication with admin/acceptance_tests/initialize_db ?

**Comment by arnaud-morvan** [comment]
This is not the standard way of doing. This is why I have a 500 lines script generated by alembic.

**Comment by arnaud-morvan** [comment]
For me it make sens to have the init db script in same package as the model, instead of multiples copies in tests.

**Comment by sbrunner** [comment]
Witch 500 lines?

**Comment by arnaud-morvan** [comment]
when you generate your alembic script using `alembic revision --autogenerate`

**Comment by sbrunner** [comment]
Then we should create a testing stuff package ...

**Comment by sbrunner** [comment]
I it generate 500 lines script It shouldn't be used ...

**Comment by arnaud-morvan** [comment]
The way I see things here is : 
- generate the database schema using initializedb to get a clean model exactly corresponding to your model.
- alter you model
- use alembic to generate a script to upgrade your database schema
This way, new database schemas exactly match with sqlalchemy model and generated script are clean.

**Pull request merged**